src/java-interop/java-interop-logger.cc

```diff
@@ -1,14 +1,18 @@
 #include <stdio.h>
 #include <stdarg.h>
 
+#ifndef _MSC_VER
+#include <strings.h>
+#endif  // ndef _MSC_VER
+
 #include "java-interop-logger.h"
 
 #define LOG_VA_ARGS(_kind_,_category_,_format_)                                 \
 	do {                                                                    \
 		const char*     kind      = (_kind_);                           \
 		LogCategories   category  = (_category_);                       \
 		va_list args;                                                   \
-		va_start (args, (_format_));                                    \
+		va_start (args, _format_);                                      \
```

Comment:
"Weird", yet required; without it, the MSVC compiler complains, because it considers `(_format_)` to be a C++ reference type, via `decltype(…)`.  This was the easiest way to bring sanity.